docgen/pom.xml

```diff
@@ -35,6 +35,21 @@
   </licenses>
 
   <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <annotationProcessorPaths>
```

Comment:
Is this missing auto-service? or is the dependency on auto-service-annotations actually a mistake?

Reply:
I think the inclusion of auto-service for docgen is incorrect. I'm not seeing any references to auto-service in any of the sources. The dependency should probably just be removed.